ts/input/tex/base/BaseConfiguration.ts

```diff
@@ -89,6 +91,36 @@ function envUndefined(_parser: TexParser, env: string) {
   throw new TexError('UnknownEnv', 'Unknown environment \'%1\'', env);
 }
 
+/**
+ * Filter for removing spacing following \nonscript
+ * @param{ParseOptions} data The active tex parser.
+ */
+function filterNonscript({data}: {data: ParseOptions}) {
+  for (const mml of data.getList('nonscript')) {
+    //
+    //  If we are in script or script-script style
+    //    remove the space (either mspace or mrow containing it)
```

Comment:
If I understand the item correctly, if the item is an `mrow` then we have added it.

Reply:
Right.  Since only actual `mspace` elements or the `mrow` elements that we added above ever get added to the `nonscript` list, if we have an `mrow`, we know it is one that we added.